starter-code/server.js

```diff
@@ -54,7 +56,7 @@ app.get('/articles', function(request, response) {
 
 app.post('/articles', function(request, response) {
   // COMMENT: What number(s) of the full-stack-diagram.png image correspond to the following line of code? Which method of article.js is interacting with this particular piece of `server.js`? What part of CRUD is being enacted/managed by this particular piece of code?
-  // Put your response here...
+  // Put your response here... Steps 3,4,5. I belive it's update since it's requesting to insert data.
```

Comment:
Which method of article.js is interacting here tho? And since you're inserting something new, its CREATE.